apps/prairielearn/src/ee/pages/instructorAiGenerateJobDetails/instructorAiGenerateJobDetails.ts

```diff
@@ -0,0 +1,30 @@
+import * as express from 'express';
+import asyncHandler from 'express-async-handler';
+
+import * as error from '@prairielearn/error';
+import { loadSqlEquiv, queryRow } from '@prairielearn/postgres';
+
+import { JobSchema } from '../../../lib/db-types.js';
+
+import { AiGenerateJobDetailsPage } from './instructorAiGenerateJobDetails.html.js';
+
+const router = express.Router();
+const sql = loadSqlEquiv(import.meta.url);
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    if (!res.locals.authz_data.has_course_permission_edit) {
+      throw new error.HttpStatusError(403, 'Access denied (must be course editor)');
+    }
+
+    const job = await queryRow(
+      sql.select_generation_job,
+      { course_id: res.locals.course.id, job_sequence_id: res.locals.lookupParams.job_id },
```

Comment:
Is this a job ID or a job sequence ID? Let's be consistent with naming and usage.

Reply:
changed in 3c8bafd1470728a8c79b488213de1dfae0aed79e